tests/ui/mut_from_ref.stderr

```diff
@@ -0,0 +1,8 @@
+error[E0261]: use of undeclared lifetime name `'b`
+  --> $DIR/mut_from_ref.rs:32:48
+   |
+32 | fn fail_double<'a>(x: &'a u32, y: &'a u32, z: &'b mut u32) -> &'a mut u32 {
+   |                                                ^^ undeclared lifetime
```

Comment:
I think you should also add a test with multiple spans now that you handle that 😄